src/main/java/com/agency04/sbss/pizza/service/CustomerService.java

```diff
@@ -0,0 +1,57 @@
+package com.agency04.sbss.pizza.service;
+
+import com.agency04.sbss.pizza.model.Customer;
+import com.agency04.sbss.pizza.exception.CustomerNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Service
+public class CustomerService {
+
+    public List<Customer> customers = new ArrayList<>(Arrays.asList(
+            new Customer("Marko1", "Marko","Horvat"),
+            new Customer("Ana1", "Ana","Kovac")));
+
+
+    public Customer getSingleCustomer(String username) {
+        for (Customer customer : customers) {
+            if (customer.getUsername().equals(username)) {
```

Comment:
Ovo ti nece dobro radit, tu ce se jedino pronaci prvi customer za sve ostale ce vratit da customer ne postoji

Reply:
Uuuups XD 
fixed it